### ISSUE TITLE
The data label of the validation set is not consistent with the validation code

### ISSUE THREAD

**Issue by Sun-WeiZhen** [stevehuanghe/SCA-CNN#8]
Hi,
This is a very excellent work for me ! Thank you a lot for providing the code in Pytorch.
 I encountered the following issues during the verification.The data label of the validation set is not consistent with the validation code.
The dataset of captions_val2014.json have four data labels('info','image','licenses','annotations'),but the code of coco.py have different labels('info','image','licenses','type'),Could you please provide the captions_val2014.json file or COCO_caption_py3 file?thank you,I downloaded the json file from COCO's official website.
